esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/quick_start.rst

```diff
@@ -0,0 +1,24 @@
+.. (C) Crown Copyright 2022, the Met Office.
+
+Quick Start
+===========
+
+.. include:: ../common.txt
+
+* Checkout the |RTW|::
+
+    git clone git@github.com:MetOffice/climate-assessment-workflow.git
+
+* Configure the |RTW|::
+
+    cd climate-assessment-workflow/climate-assessment-workflow
+    rose edit
+
+* Run the |RTW| at the Met Office, where ``<run-name>`` is a unique run name
+  relevant to the current configuration::
+
+    cylc install --run-name=<run-name> -O metoffice
+    cylc play climate-assessment-workflow/<run-name>
```

Comment:
Update these to not refer to `climate-assessment-workflow`.

Reply:
a75dc13cd done, although I wasn't sure if the cylc command was cylc play recipe_test_workflow.